From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Thu, 13 Dec 2018 09:50:46 -0800
Subject: [PATCH] Use pdns uid/gid, and enable syslogging.

By default, pdns would run as root. We could code this as default, but
this is relatively simple and can be overridden easily as well.

Syslogging is nicer than not, let the journal collect the output.
---
 pdns/pdns.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pdns/pdns.service.in b/pdns/pdns.service.in
index 3c43b07..cbcb670 100644
--- a/pdns/pdns.service.in
+++ b/pdns/pdns.service.in
@@ -6,7 +6,7 @@ Wants=network-online.target
 After=network-online.target mysqld.service postgresql.service slapd.service mariadb.service
 
 [Service]
-ExecStart=@sbindir@/pdns_server --guardian=no --daemon=no --disable-syslog --log-timestamp=no --write-pid=no
+ExecStart=@sbindir@/pdns_server --guardian=no --daemon=no --log-timestamp=no --write-pid=no
 SyslogIdentifier=pdns_server
 User=@service_user@
 Group=@service_group@
